interface/settings.py

```diff
@@ -0,0 +1,21 @@
+from .base_settings import * # noqa
```

Comment:
I'm starting to think that the pattern of `base_settings` and `settings` is overkill, a single file may be enough. The testsuite can import the main settings file and override whatever it needs.